frontend/src/App.test.tsx

```diff
@@ -1,7 +1,27 @@
 import React from "react";
-import { render } from "@testing-library/react";
-import App from "./App";
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter as Router } from "react-router-dom";
+import Navbar from "./components/navbar";
 
-test("renders learn react link", () => {
-  render(<App />);
+test("renders all links", () => {
+  render(
+    <Router>
+      <Navbar />
+    </Router>,
+  );
+
+  const homeLink = screen.getByText(/HOME/i);
+  expect(homeLink).toBeInTheDocument();
```

Comment:
Don't have to change anything here but just wanted to point out an easier shortcut.

you can get getByText from render

so you can do const { getByText } = render ( <Router> <Navbar /> </Router>)